src/test/java/it/com/atlassian/bitbucket/jenkins/internal/config/BitbucketPluginConfigurationIT.java

```diff
@@ -80,10 +80,6 @@ public void testAddBitbucketServer() throws Exception {
         waitTillItemIsRendered(adminCredential::getOptions);
         adminCredential.getOption(1).click();
 
-        HtmlSelect credential = form.getSelectByName("_.credentialsId");
```

Comment:
@mhenschke-atl I believe this was missed when you removed the `credentialsId` from the config screen. It now breaks so I removed it. If that was in-correct let me know and I'll re-instate and work out why it can't find the `credentialsId`

Reply:
Good catch, thanks for fixing. Yes the credentialsId has been removed entirely from the plugin page so this should be correct to remove.